app/grandchallenge/archives/models.py

```diff
@@ -359,6 +359,13 @@ def delete_url(self):
             kwargs={"slug": self.base_object.slug, "pk": self.pk},
         )
 
+    @property
+    def detail_url(self):
```

Comment:
This should be called `get_absolute_url`, see https://docs.djangoproject.com/en/5.0/ref/models/instances/#get-absolute-url